### DISCUSSION TITLE
Is it possible to hide the bottom navigation?

### DISCUSSION THREAD

**Discussion by anniekao** [storybookjs/react-native#624] [Q&A] [answered]
Hi there,

My team recently migrated to Storybook 8 and really appreciate the UI changes. We've recently started to use Storybook to layout screens for our app, however we were wondering if there was a way to hide the bottom nav and have a full screen option or if there's an option that we missed while going through the docs.

Here's an example from one of our screen components, you can see there's more content, but it's being obscured by the Storybook nav.

![Screenshot 2024-10-20 at 3 25 04 PM](https://github.com/user-attachments/assets/14858b51-105f-4a4d-a3b3-a10cfa6c2ba2)

**Comment by dannyhw** [comment, marked as answer]
there is the ondeviceui option where you can completely turn off the ui but I haven't currently got a way to temporarily hide the nav other than that. I could add it but need to think of a good way to fit that button in somewhere

**Reply by anniekao** [reply]
Thanks for the quick reply (on a Sunday no less)! It works as an interim solution.
A temporary hide option would be amazing, but totally understandable that figuring out placement is tricky.

**Reply by dannyhw** [reply]
How about this as a solution?
https://github.com/storybookjs/react-native/pull/632

**Reply by anniekao** [reply]
It looks really good! I liked the suggestion about having the button draggable in the MR. Anything is better than setting/unsetting onDeviceUI. Thanks for getting on top of this so quickly!
